go/cmd/vtclient/vtclient.go

```diff
@@ -141,6 +142,9 @@ func run() (*results, error) {
 	flag.Parse()
 	args := flag.Args()
 
+	closer := trace.StartTracing("vtclient")
```

Comment:
This is not starting a span, it's starting tracing on the current process.

Reply:
Starting tracing seems like something `go/cmd/vtclient/plugin_opentracing.go` should do, for consistency with how we handle metrics (`go/cmd/vtctld/plugin_prometheusbackend.go` would be an example)

The tricky thing here seems like the closing, since there's not yet any `go/vt/clientenv` yet as a parallel to `go/vt/servenv` which could let a plugin register shutdown hooks across all the client binaries.

It looks like most vitess client binaries do `defer logutil.Flush()` and `defer exit.RecoverAll()` or `exit.Recover()` (from `go/exit`) at the beginning.

Probably the "right" thing to do is to create a more formal `go/vt/clientenv` with pluggable exit hooks, but for the short term maybe just having `defer trace.Flush()` or `defer trace.Close()` or something like could be fine.